src/includes/enriching-events/set-user/apple.mdx

```diff
@@ -14,4 +14,8 @@ user.email = @"john.doe@example.com";
 [SentrySDK setUser:user];
 ```
 
-Since 6.0.1: Sentry will by fallback to `installationId` if you do not provide a user identity.
+<Note>
+
+Since 6.0.1: If you do not provide a user identity, the SDK falls back to `installationId`, which the SDK once randomly generates during an app's installation.
```

Comment:
```suggestion
Since 6.0.1: If you do not provide a user identity, the SDK falls back to `installationId`, which the SDK randomly generates once during an app's installation.
```